<?xml version="1.0" ?>
<test projectname='jabref-gui-test' reportdir='ju-2012-02-29-16-44-43' >
<testsuite name="SelectedTests" >
  <testcase name="test" status="2" time="22.096" >
<![CDATA[net.sourceforge.marathon.junit.MarathonAssertion: Application under test aborted - use set_no_fail_on_exit to suppress this error
	Failure: Application under test aborted - use set_no_fail_on_exit to suppress this error
	at System.exit(internal:1)
	at net.sourceforge.marathon.junit.MarathonTestCase.confirmResult(MarathonTestCase.java:191)
	at net.sourceforge.marathon.junit.MarathonTestCase.runTest(MarathonTestCase.java:135)
  ]]></testcase>
</testsuite>
</test>